Source/Scene/Expression.js

```diff
@@ -670,6 +687,13 @@ define([
         };
     }
 
+    function getEvaluateTernaryFunction(call) {
+        var evaluate = ternaryFunctions[call];
+        return function(feature) {
+            return evaluate(this._left.evaluate(feature), this._right.evaluate(feature));
```

Comment:
This should fix the errors here: https://github.com/AnalyticalGraphicsInc/cesium/pull/4709/files#r91228640